cp-url.go

```diff
@@ -31,6 +31,15 @@ type copyURLs struct {
 	Error         *probe.Error `json:"-"`
 }
 
+func (m *copyURLs) Equal(o interface{}) bool {
```

Comment:
Same here why would you take an interface? when one can avoid compile time problems with proper types?